actors/miner/src/monies.rs

```diff
@@ -267,10 +270,30 @@ pub fn initial_pledge_for_power(
     let lock_target_denom = LOCK_TARGET_FACTOR_DENOM;
     let pledge_share_num = qa_power;
     let network_qa_power = network_qa_power_estimate.estimate();
-    let pledge_share_denom = cmp::max(cmp::max(&network_qa_power, baseline_power), qa_power);
+
+    // Compute gamma based on current_epoch
```

Comment:
where can we write a unittest for the updated initial_pledge_for_power so that we can be more certain that the computation is what we expect?

Reply:
just add a new file in actors/miner/tests/, you'll see it already being tested for a particular case in actors/miner/tests/precommit_deposit_and_initial_pledge_positive_test.rs so you can copy that pattern